packages/manager/apps/procedures/src/pages/create/LegalInformations.tsx

```diff
@@ -7,17 +7,25 @@ import {
 } from '@ovhcloud/ods-common-theming';
 import { ODS_TEXT_SIZE } from '@ovhcloud/ods-components';
 import useUser from '@/context/User/useUser';
-import { LegalPolicyLinkByLanguage } from './create.constants';
+import { LegalPolicyLinkByLanguage } from '@/constants';
 
 export const LegalInformations: FunctionComponent = () => {
-  const { t } = useTranslation('account-disable-2fa');
-  const { user } = useUser();
+  const {
+    t,
+    i18n: { language },
+  } = useTranslation('account-disable-2fa');
+  const {
+    user: { subsidiary },
+  } = useUser();
 
-  const legalPolicyLink =
-    (LegalPolicyLinkByLanguage as any)[user.language] ||
-    LegalPolicyLinkByLanguage.DEFAULT;
+  const legalPolicyLink: string =
+    subsidiary === 'CA'
+      ? (LegalPolicyLinkByLanguage[subsidiary] as any)[language] ||
```

Comment:
```suggestion
      ? (LegalPolicyLinkByLanguage[subsidiary])[language as key in Subsidiary] ||
```

It's also Typescript correct and it's more readable I think for a external reader